packages/malloy/src/lang/ast/field-space/dynamic-space.ts

```diff
@@ -133,7 +133,7 @@ export abstract class DynamicSpace extends StaticSpace {
       for (const [, field] of reorderFields) {
         if (field instanceof JoinSpaceField) {
           const joinStruct = field.join.structDef(parameterSpace);
-          if (!ErrorFactory.isErrorStructDef(joinStruct)) {
+          if (!ErrorFactory.fromErrorFactory(joinStruct)) {
```

Comment:
`ErrorFactory.fromErrorFactory` is a bit of a confusing name. I think I liked `isErrorStructDef` better...

Reply:
I imagine error factory returning many error types. changed it to `ErrorFactor.didCreate()`